web/src/app/wizard/WizardScheduleForm.js

```diff
@@ -36,34 +37,24 @@ const styles = {
  */
 @withWidth()
```

Comment:
Instead of this decorator/HOC injecting the `width` prop into this class, you can use the `useWidth` we have created from the util directory:

```
import useWidth from '../../util/useWidth'
```
and
```
const width = useWidth()
```

Reply:
Alright! Thank you